src/styles/blocks/nav.scss

```
@@ -0,0 +1,43 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding: 23px 0;
+
+    font-size: $font-size-2;
+    color: $color_dark;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link:hover {
+    color: $color-blue;
+  }
+}
+
+.is-active {
+  color: $color-blue;
+}
+
+.is-active::after {
+  position: absolute;
+  width: 37px;
```

I think hardcoding width  only for 1 case with 1st link is a mistake. 
```suggestion
  width: 100%;
```